app/models/firm.rb

```diff
@@ -85,6 +85,28 @@ def subsidiary?
     parent.present?
   end
 
+  def field_order
+    [
+      :email_address,
+      :telephone_number,
+      :address_line_one,
+      :address_line_two,
+      :address_town,
+      :address_county,
+      :address_postcode,
+      :in_person_advice_methods,
+      :free_initial_meeting,
+      :initial_meeting_duration,
+      :initial_advice_fee_structures,
+      :ongoing_advice_fee_structures,
+      :allowed_payment_methods,
+      :minimum_fixed_fee,
+      :percent_total
+    ] +
```

Comment:
Yeah, agreed, it's not very pretty.  I'll have a play with some approaches along the lines of what you suggest above.